### ISSUE TITLE
Access level for charts

### ISSUE THREAD

**Issue by SnarfW** [vesta-webtrees-2-custom-modules/vesta_extended_relationships#119]
I would like to hide the charts in the 'more charts menu' for visitors. 
According to the instructions this should be possible via Control Panel > Charts
Unfortunately, the charts are always shown, even when I set the access level to 'Hide from everyone'. Is this a bug, or is something in my set-up hampering this to function correct? 
I also see only one menu, I was expecting to see an option to set the access per chart. 

![afbeelding](https://github.com/vesta-webtrees-2-custom-modules/vesta_extended_relationships/assets/25879600/5da5412b-7e4b-4955-bf95-12fb91906f97)

I did now hide them all by setting the access level for the menu (Control Panel > Menu), which is working as expected

Using vesta  2.1.17.0.3
Webtrees 2.1.17

**Comment by ric2016** [comment]
These options are only for the original relationships chart.

The new charts are experimental and will be fully customizable as soon as webtrees supports container modules, see [here](https://github.com/fisharebest/webtrees/issues/4795) .

Unfortunately there is no progress at all on this issue (and many others), and no indication whether this will change. It may help if you leave a comment for that issue, but I'm not optimistic that further custom module support is on the roadmap at all.

**Comment by SnarfW** [comment]
You mentioned that the new charts are experimental, I wasn't aware. They are include in your 'stable' release zip-file from your homepage. Wouldn't it be better not to include them (yet) in the zip?

I also see that there are many open issue for webtrees...

**Comment by kiwi3685** [comment]
As a temporary fix, to enable removing the "More charts" menu item while keeping the original relationships chart available, could you just add a new class to the menu item.

Like:
        return new Menu(
            I18N::translate('More Charts'),
            '#',
            'menu-chart more-charts',
            ['rel' => 'nofollow'],
            $submenus->all());

Then "display:none; can be added in css by anyone who wants this option. Perhaps preferable than you removing these charts completely.

**Comment by ric2016** [comment]
Please [support the container modules issue](https://github.com/fisharebest/webtrees/issues/4795) in webtrees to get this fixed properly. 

You don't need css to hide the extra menu though: Just set its access level in the control panel to "hide from anyone". It may seem, due to the name, that this would affect the entire module, but it actually only affects the menu part.

**Comment by kiwi3685** [comment]
I trued that before I made the suggestion. It also removed the Relationship chart link from the main Charts menu group. 
I can check it again tomorrow if you’re sure it should work.

**Comment by ric2016** [comment]
Yes, this work, I just re-checked myself. Make sure to set the access level under 
Modules > Genealogy > Menus > Vesta Extended Relationships (and keep 'Enabled' checked).

**Comment by kiwi3685** [comment]
Thank you. Now I see the problem.

I was changing access to the item "Modules > Genealogy > **Charts** > Vesta Extended Relationships".

I should have known better.

**Comment by ric2016** [comment]
Well it is a bit confusing because the extra menu also contains charts.
It really would be much simpler with container modules.
